### DISCUSSION TITLE
Windows 9x: must press a key before mouse double-click works?

### DISCUSSION THREAD

**Discussion by emendelson** [joncampbell123/dosbox-x#4422] [Q&A] [answered]
I've been working on a DOSBox-X-based Windows 98 SE system that I hope may be of interest to other people, and I've come across one strange issue:

When I start Windows 98, with autolock=on in the config file, I can click in the window in order to capture the mouse. The mouse pointer than moves around the Windows 98 desktop, as expected, but nothing happens when I double-click on a desktop icon - until I press a key (any key). After I press a key, I can double-click on a desktop icon and the icon will respond.

Oddly, I can click on the Start Menu before pressing a key, and the Start Menu will open. But I can't activate any desktop icons until after I press a key.

Is there any way to avoid this key-press requirement?

**Comment by rderooy** [comment, marked as answer]
Do you have turbo mode engaged? That would explain the behaviour, with Turbo mode you cannot double-click fast enough. This is mentioned on the Wiki page for installing Win9x.

A key press will however disengage Turbo mode.

**Reply by emendelson** [reply]
That is exactly what is happening - and I did read that page and should have figured it out for myself. Thank you.

**Reply by Torinde** [reply]
can you mark this discussion as "Answered"?

**Reply by Torinde** [reply]
It seems discussions, depending on their type, can be both "Answered" and "Closed". This one is answered, but not closed. Not sure if it should be, but just mentioning that (e.g. to get the list of open topics shorter).

**Comment by emendelson** [comment]
Done!

**Reply by Torinde** [reply]
Thanks!
I see two other "answered, but not closed" - again, I'm not sure if/when those should be closed:
![image](https://github.com/joncampbell123/dosbox-x/assets/97228894/8ab05af0-057b-429e-a49e-641c35ae6f14)
